doc/source/merging.rst

```diff
@@ -310,7 +310,8 @@ Concatenating with mixed ndims
 
 You can concatenate a mix of Series and DataFrames. The
 Series will be transformed to DataFrames with the column name as
-the name of the Series.
+the name of the Series. To concatenate only one Series it is preferable
```

Comment:
Can you show a mini-example inline here.

Reply:
> Can you show a mini-example inline here.

You mean a full example like this one (not fit for our case of course)?

```
.. ipython:: python

   s1 = pd.Series(['X0', 'X1', 'X2', 'X3'], name='X')
   result = pd.concat([df1, s1], axis=1)

.. ipython:: python
   :suppress:

   @savefig merging_concat_mixed_ndim.png
   p.plot([df1, s1], result,
          labels=['df1', 's1'], vertical=False);
   plt.close('all');
```

If so, looking at the broader context of that area of the document: the use of `pd.concat()` I feel like this would confuse the user more. We are in a `pd.concat()` context for several examples in a row and this would introduce a single `df.assign()` call example in between them. An in-paragraph example would be ok for me. Otherwise, I think we should have a title and cover the use case completely separately (right now it is under the *Concatenating with mixed ndims* title).

In any case, I have no strong opinion, just let me know how you want it done.

Reply:
yes. you can make a small sub-section for this.